### PR TITLE
Add Jellyfin.

### PR DESCRIPTION
**Pull request by Lin-Youth**
### Types of Changes

**What types of changes does your PR introduce?** Put an x in all boxes that apply

- [x ] New addition to list

### Proposed Changes

**Describe each of your changes here to communicate to the maintainers why we should accept this pull request.**

1. Add Jellyfin.

### PR Checklist

Put an x in the boxes once you've completed each step. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.

- [ x] I have read the CONTRIBUTING guide
- [ x] I have explained why this PR is important
- [ x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1058,9 +1059,10 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
...
 * [DisplayBuddy](https://displaybuddy.app) - Control the brightness, contrast, input source and more of your external display directly from your Mac.
 * [f.lux](https://justgetflux.com/) - Makes the color of your computer's display adapt to the time of day. ![Freeware][Freeware Icon]
-* [Lunar](https://lunar.fyi/) -  Help you adujst brightness, contrast and volumn of your external display. [![Open-Source Software][OSS Icon]](https://github.com/alin23/Lunar) ![Freeware][Freeware Icon]
 * [Grayscale Mode](https://github.com/rkbhochalya/grayscale-mode) - An open source macOS app that lets you quickly toggle grayscale filter right from your menu bar or using a keyboard shortcut (⌥⌘G). [![Open-Source Software][OSS Icon]](https://github.com/rkbhochalya/grayscale-mode) ![Freeware][Freeware Icon]
+* [IconChanger](https://github.com/underthestars-zhy/IconChanger) - Simply change your app's icon on macOS. Just a click. [![Open-Source Software][OSS Icon]](https://github.com/underthestars-zhy/IconChanger) ![Freeware][Freeware Icon]
```
Comment (jaywcjlove) [inline README.md:1063]: @Lin-Youth Nice tool, it doesn't appear to be a desktop application.

Comment (Lin-Youth) [reply README.md:1063]: I put a screenshot of the desktop app running on my mac below: 
image

Comment (jaywcjlove) [reply README.md:1063]: I can't find the app download link for it

@Lin-Youth

**Comment by Lin-Youth** [comment]
Release page：https://github.com/underthestars-zhy/IconChanger/releases

File: README.md (hunk 1 above)
Comment (Lin-Youth) [reply README.md:1063]: Just at the release page：https://github.com/underthestars-zhy/IconChanger/releases

Comment (jaywcjlove) [reply README.md:1063]: you can remove it, thanks

Comment (Lin-Youth) [reply README.md]: done. Only Jellyfin added.

**Pull request merged**